common/tools/dev-tool/src/commands/run/extract-api.ts

```diff
@@ -0,0 +1,130 @@
+import { leafCommand, makeCommandInfo } from "../../framework/command";
+import {
+  Extractor,
+  ExtractorConfig,
+  ExtractorResult,
+  IConfigApiReport,
+  IConfigDocModel,
+  IConfigDtsRollup,
+  IConfigFile,
+} from "@microsoft/api-extractor";
+
+import { createPrinter } from "../../util/printer";
+import { resolveProject } from "../../util/resolveProject";
+import path from "path";
+import { readFile } from "fs-extra";
+
+export const commandInfo = makeCommandInfo(
+  "extract-api",
+  "Runs api-extractor multiple times for all exports.",
+  {}
+);
+
+const log = createPrinter("extract-api");
+
+function getSuffix(exportPath: string): string {
+  return exportPath.replaceAll("/", "\u2215"); // replace with division slash to avoid file system issue
```

Comment:
interesting that Windows doesn't allow adding files into zip when the filename contain division slash.